docs/en/user_guides/deploy.md

```diff
@@ -1,4 +1,212 @@
-# Model Deployment
+# MMDetection Deployment
 
-MMDet 3.x fully relies on [MMDeploy](https://mmdeploy.readthedocs.io/) to deploy models.
-Please stay tuned and this document will be update soon.
+- [MMDetection Deployment](#mmdetection-deployment)
+  - [Installation](#installation)
+    - [Install mmdet](#install-mmdet)
+    - [Install mmdeploy](#install-mmdeploy)
+  - [Convert model](#convert-model)
+  - [Model specification](#model-specification)
+  - [Model inference](#model-inference)
+    - [Backend model inference](#backend-model-inference)
+    - [SDK model inference](#sdk-model-inference)
+  - [Supported models](#supported-models)
+
+______________________________________________________________________
+
+[MMDetection](https://github.com/open-mmlab/mmdetection) aka `mmdet` is an open source object detection toolbox based on PyTorch. It is a part of the [OpenMMLab](https://openmmlab.com/) project.
```

Comment:
It seems that we do not need to introduce mmdet anymore, but we need to refer to the doc in mmdeploy to make users know that they could refer to mmdeploy for the most recent doc.

Reply:
done